### ISSUE TITLE
[question] folding type=marker

### ISSUE THREAD

**Issue by ninrod** [syl20bnr/spacemacs#6962]
Is there a way to define a commented marker to mark a folding region in spacemacs like so? 

```sh
# this is the fold start {{{
some random code
blah 
blah
# }}} <- this is where the fold ends
```

**Comment by TheBB** [comment]
No good way to do this currently. Hideshow supports it but I haven't been able to figure out how to do it. There's an issue open to [vimish-fold](https://github.com/mrkkrp/vimish-fold/issues/30) so as soon as someone feels the need, it will land there eventually. Maybe you can do it. :-)

**Comment by nixmaniack** [comment]
It's mentioned in [origami's readme](https://github.com/gregsexton/origami.el#does-it-support-my-favourite-major-mode) that triple braces are supported. So if you use `origami`  folding method by setting `dotspacemacs-folding-method` variable, it should work. Not sure how elegantly it works though as I don't use them.

**Comment by TheBB** [comment]
Ah thanks, that's news to me. So this works then, all you have to do is use origami folding and set `origami-fold-style` to `triple-braces` in the buffers where you want it. E.g.

```elisp
(add-hook 'prog-mode-hook (lambda () (setq-local origami-fold-style 'triple-braces))
```

for all programming modes. (Note it's not enough to `setq-default`, it has to be buffer-local.)

Origami is supported on develop, so I'm marking this as fixed in develop.

**Comment by ninrod** [comment]
nice. I'll test when I get home. So this must be what I'd have to do, right? 

https://github.com/gregsexton/origami.el#can-i-override-the-folding-parser-for-an-individual-file

**Comment by TheBB** [comment]
That's one way, but any other way of setting buffer-local variables will do as well. For example directory-local variables or the solution I gave in my previous post.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
